projects/netgrif-components-core/src/lib/view/task-view/abstract-single-task-view.component.ts

```diff
@@ -18,22 +17,25 @@ export abstract class AbstractSingleTaskViewComponent extends AbstractViewWithHe
 
     @Input() initiallyExpanded: boolean = true;
     @Input() preventCollapse: boolean = true;
-    public task$: Observable<TaskPanelData>;
+    public taskPanelData: BehaviorSubject<TaskPanelData>;
     public loading$: Observable<boolean>;
     private transitionId: string;
     private subRoute: Subscription;
-    protected unsubscribe$: Subject<void>;
+    protected subPanelData: Subscription;
 
     protected constructor(protected taskViewService: TaskViewService,
                           activatedRoute: ActivatedRoute) {
         super(taskViewService, activatedRoute);
-        this.unsubscribe$ = new Subject<void>();
+        this.subPanelData = new Subscription();
```

Comment:
why? It would be better to define `subPanelData` as `Subscription | undefined`

Reply:
Why?

Reply:
what does `new Subscription()` even do?

Reply:
using `Subscription.EMPTY` might be better - https://rxjs.dev/api/index/class/Subscription#static-properties